### ISSUE TITLE
Script import error not shown

### ISSUE THREAD

**Issue by 1AmYF** [piqueserver/piqueserver#535]
The issue is already indicated here
https://github.com/piqueserver/piqueserver/blob/75dfc300384070d793f1e0dde1aa5a40b5b513db/piqueserver/extensions.py#L75
This means that piqueserver will report something like
>[piqueserver.extensions#error] ('script xyz' not found: ModuleNotFoundError("No module named 'xyz'",))

which is the same error as if the file was not found. Would be nice to see the actual import error of the script instead.

**Comment by NotAFile** [comment]
I'm suprised we don't already have an issue for this.

This is a limitation of the `imp` api we are using, which is deprecated in py3 anyway. The solution is porting to the python 3 `importlib` module. Unlike `imp`, the newer API allows us to separate errors caused when finding the file from errors caused when loading the file.

**Comment by godwhoa** [comment]
Took a jab at this got it working
```
Loading config from 'piqueserver/config/config.toml'
2019-08-18T00:54:39+0530 [piqueserver.extensions#error] ('script lolidontexist' not found in either piqueserver/config/scripts/ directory or global scope)
2019-08-18T00:54:39+0530 [piqueserver.extensions#error] Error while loading script piqueserver.scripts.afk: ModuleNotFoundError("No module named 'xyz'")
```
Should it exit if a script is not loaded properly or just log it?

**Comment by godwhoa** [comment]
Closed with: #544

**Comment by NotAFile** [comment]
`piqueserver/map.py` still has some imp
